packages/paypal-commerce-integration/src/paypal-commerce-credit-card/paypal-commerce-credit-cards-payment-strategy.spec.ts

```diff
@@ -566,6 +572,34 @@ describe('PayPalCommerceCreditCardsPaymentStrategy', () => {
                 },
             });
         });
+
+        it('submits hosted form without billing address if there is vaulted form', async () => {
+            const cardFields = paypalSdk.CardFields(
+                cardFieldsOptionsMock as PayPalCommerceCardFieldsConfig,
+            );
+            const optionsWithVaultingForm = {
+                ...initializationOptions,
+                paypalcommercecreditcards: {
+                    ...paypalCommerceCreditCardsOptions,
+                    form: {
+                        fields: creditCardVaultedForm,
+                    },
+                },
+            } as PaymentInitializeOptions;
+
+            const submitMock = jest.fn().mockReturnValue(Promise.resolve());
+
+            jest.spyOn(paypalSdk, 'CardFields').mockImplementation(() => ({
+                ...cardFields,
+                submit: submitMock,
+            }));
+
+            await strategy.initialize(optionsWithVaultingForm);
+
+            await strategy.execute(defaultExecutePayload);
+
+            expect(submitMock).toHaveBeenCalledWith({});
```

Comment:
It is not necessary to send empty object

Reply:
Updated